src/darwin/Framework/CHIP/MTRDeviceConnectivityMonitor.h

```diff
@@ -0,0 +1,29 @@
+/**
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
```

Comment:
Could we add a few lines describing what the MTRDeviceConnectivityMonitor does? That would be very helpful.

Reply:
Yes please. Ideally on the `@interface` declaration.

Reply:
Done